docs/source/configuration/authorization.mdx

```diff
@@ -98,199 +139,264 @@ So the `me` top level operation would not even be executed. When returning the r
 
-Assuming we have this schema:
+- An authenticated user's profile (`me`)
+- Other user's emails
+- Products and the users who have ordered it
+
+Your schema may look something like this:
 
 ```graphql
 type Query {
-    me: User @requiresScopes(scopes: ["profile:read"])
-    products: [Products]
+  me: User @requiresScopes(scopes: ["profile:read"])
+  products: [Products]
 }
 
 type User {
-    id: ID
-    name: String
-    email: @requiresScopes(["user:read"])
+  id: ID
+  name: String
+  email: @requiresScopes(["user:read"])
 }
 
 type Product {
-    id: ID
-    name: String
-    amount: Int @requiresScopes(["employee", "inventory"])
-    orders: [User] @requiresScopes(["employee"])
+  id: ID
+  name: String
+  amount: Int @requiresScopes(["inventory:read"])
+  orders: [User] @requiresScopes(["user:read", "inventory:read"])
 }
 ```
 
-And this query:
+The router executes the following query differently, depending on the request's attached scopes:
 
 ```graphql
-{
-    me {
-        name
-        email
-    }
+query {
+  me {
+    name
+    email
+  }
 
-    products {
-        name
-        amount
-        orders {
-            name
-        }
+  products {
+    name
+    amount
+    orders {
+      name
     }
+  }
 }
 ```
 
-If the request presented the scopes `profile:read user:read`, then it would be interpreted as:
+If the request includes the `profile:read user:read` scope set, then the router would execute the following filtered query: 
 
+```graphql title="Request scopes: 'profile:read user:read'"
+query {
+  me {
+    name
+    email
+  }
 
-```graphql
-{
-    me {
-        name
-        email
-    }
+  products {
+    name
+    /* The following fields are filtered out:
+    amount # Requires "inventory:read" scope
+    orders # Requires "user:read" and "inventory:read" scopes
+    */
+  }
+}
+```
+
+The response would include errors at the `/products/@/amount` and `/products/@/amount` paths.
+
+If the request includes the `user:read inventory:read` scope set, then the router would execute the following filtered query: 
 
-    products {
-        name
+```graphql title="Request scopes: 'user:read inventory:read'"
+query {
+  /* The following fields are filtered out
+  me { # Requires the "profile:read" scope
+    name
+    email
+  }
+  */
+  products {
+    name
+    amount
+    orders {
+      name
     }
+  }
 }
 ```
 
-And generate an error at path `/products/@/amount`.
+The response would include an error at the `/me` path.
 
-While if it presented the scopes `employee inventory`, it would filter the query as:
+## Composition and federation
 
-```graphql
-{
-    products {
-        name
-        amount
-        orders {
-            name
-        }
-    }
+Authorization directives are defined at the subgraph level and the router composes them in the supergraph schema. In other words, if subgraph fields or types include either `@authenticated` or `@requiresScopes` directives, they are set on the supergraph too.
```

Comment:
As noted in another comment - either rover or studio that is responsible for composition